maat-court-data-api/src/main/java/gov/uk/courtdata/validator/MaatIdValidator.java

```diff
@@ -38,4 +41,21 @@ public Optional<Void> validate(final Integer maatId) {
         }
 
     }
```

Comment:
**A nice to have** Could you rename the existing method to something more descriptive?

i.e. 
`    public Optional<Void> validate(final Integer maatId) {`
to
`    public Optional<Void> validateExists(final Integer maatId) {
`

Reply:
I've decided to leave it as-is because there are several other validator classes within the same folder that are structured in the same way (with a single validate method), so have tried to keep as consistent as I can. I may have validated single responsibility by adding the 'validateNotExists' method to this class, but I feel it makes sense to allow these classes to do a bit more than just have a single validate method. There may be opportunities for some refactoring here on some of these other classes.

Reply:
yeah no worries, makes sense :)